utils/provided-goi-analysis-functions.R

```diff
@@ -6,34 +6,35 @@
 #
 # source(file.path("utils", "provided-goi-analysis-functions.R"))
 
-colnames_to_gene_symbols <- function(normalized_sce_matrix,
+colnames_to_plotting_symbols <- function(normalized_sce_matrix,
                                     goi_list,
-                                    ensembl_id_column,
-                                    gene_symbol_column) {
+                                    sce_rownames_column,
+                                    optional_plotting_column) {
   # Given a normalized SingleCellExperiment matrix, the name of the column with
-  # the Ensembl gene identifiers, the name of the associated gene symbols,
-  # convert the matrix column names into symbols for heatmap plotting.
+  # the SCE rowname identifiers, the name of the associated plotting identifiers,
+  # convert the matrix column names into the plotting ids for heatmap plotting.
   #
   # Args:
   #   normalized_sce_matrix: matrix retrieved from a normalized
   #                          SingleCellExperiment object
   #   goi_list: data frame with provided genes of interest relevant to the data
   #             in the SingleCellExperiment object
-  #   ensembl_id_column: name of the column with the ensembl gene identifiers
-  #   gene_symbol_column: name of the column with the gene symbols that would be
-  #                       used for plotting if provided at the command line
+  #   sce_rownames_column: name of the column with the SCE row identifiers
+  #   optional_plotting_column: name of the column with the identifiers that
+  #                             would be used for plotting if provided at the 
+  #                             command line
```

Comment:
```suggestion
  #   plotting_column: name of the column with the identifiers to be used for plotting
```
Since you don't call this function from the command line and this isn't optional here.